Data-Engineer.md

```diff
@@ -0,0 +1,63 @@
+# Work Sample for Data Engineer
+
+Solve the following data pipeline problems using a DAG (Directed Acyclic Graph) oriented tool, such as Pachyderm, Airflow, Dagster, etc.
+
+## Problem 1: Raw Data Processing
+
+**Objective**: Ingest and process raw stock market datasets.
+
+### Tasks:
+1. Download the raw datasets from https://www.kaggle.com/datasets/jacksoncrow/stock-market-dataset.
```

Comment:
There's the primary dataset on display and then the thousands of sub-datasets. So I would change the wording to specifically say ETF & Stock datasets in this first line in addition to step 2